### ISSUE TITLE
MA0007 is not triggered in multiline pattern-matching expressions

### ISSUE THREAD

**Issue by efpies** [meziantou/Meziantou.Analyzer#996]
### Version of the Meziantou.Analyzer NuGet package

2.0.286

### Rule Identifier

MA0007

### Target Framework

net8.0

### C# Language version

C# 12

### Description

Resharper's analyzer shows that a comma should be added after the last clause in a multiline `is { }` expression, but Meziantou Analyzers don't, so it can't be enforced during build.

### Reproduction Steps

Minimal code:

```c#
var res = contract is
{
    One: false,
    Two: false
};
```

**Comment by meziantou** [comment]
The latest version (available soon: https://github.com/meziantou/Meziantou.Analyzer/actions/runs/21685046246) should fix the issue.
